chore/travis/travis-jdk10.sh

```diff
@@ -8,6 +8,6 @@ wget https://raw.githubusercontent.com/sormuras/bach/master/install-jdk.sh
 chmod +x install-jdk.sh
 
 export JAVA_HOME=$HOME/openjdk8
-source ./install-jdk.sh -f 10
+source ./install-jdk.sh -f 10 -c
```

Comment:
Could you explain why you needed this argument?

Reply:
Yes! So basically it tells `install-jdk.sh` to use the default (the one present in the base conatiner) collection of trusted certificate authority (CA) certificates for java. Without this option, calling maven from java results in SSL errors.

Reply:
ok, can you add a comment on the script please?

Reply:
Done.